.github/workflows/formatting.yml

```diff
@@ -0,0 +1,42 @@
+name: Code Formatting Check
+
+on:
+  push:
+    branches: [develop, "release/*"]
+  pull_request:
+    branches: ["*"]
+
+env:
+  # Please ensure that this is in sync with graalVersion in build.sbt
+  graalVersion: 21.3.0
+  # Please ensure that this is in sync with javaVersion in build.sbt
+  javaVersion: 11
+  # Please ensure that this is in sync with project/build.properties
+  sbtVersion: 1.5.2
```

Comment:
we have `1.6.1` in biuld properties. I think it's worth updating it for all workflows

Reply:
Yes, I basically wanted c&p existing workflows.